File: aat-lib/src/main/java/ch/bailu/aat_lib/xml/parser/gpx/WptParser.java
package ch.bailu.aat_lib.xml.parser.gpx;

import org.xmlpull.v1.XmlPullParser;

import java.io.IOException;

import ch.bailu.aat_lib.gpx.GpxConstants;
import ch.bailu.aat_lib.xml.parser.scanner.Scanner;

public class WptParser extends PntParser {


    public WptParser() {
        super(GpxConstants.QNAME_WAY_POINT);
    }


    @Override
    protected void parsed(XmlPullParser parser, Scanner scanner) throws IOException {
        scanner.wayParsed.onHavePoint();
    }
}

File: aat-lib/src/main/java/ch/bailu/aat_lib/file/xml/parser/gpx/WptParser.kt
package ch.bailu.aat_lib.file.xml.parser.gpx

import ch.bailu.aat_lib.gpx.GpxConstants
import ch.bailu.aat_lib.file.xml.parser.scanner.Scanner
import org.xmlpull.v1.XmlPullParser
import java.io.IOException

class WptParser : PntParser(GpxConstants.QNAME_WAY_POINT) {
    @Throws(IOException::class)
    override fun parsed(parser: XmlPullParser, scanner: Scanner) {
        scanner.wayParsed.onHavePoint()
    }
}
